spec-main/api-safe-storage-spec.ts

```diff
@@ -0,0 +1,24 @@
+import { expect } from 'chai';
```

Comment:
Can we add tests for:

* Passing a non-buffer to decryptString
* Passing UTF-16 characters to encryptString and asserting that can be decrypted again
* Passing a buffer to decryptString that is not actually encrypted data (an invalid ciphertext)